build/images/scripts/install_cni

```diff
@@ -2,25 +2,38 @@
 
```

Comment:
I wonder if in the future we should consider replacing this bash script with a small Go binary for maintainability and unit testing?